server/mess_server.py

```diff
@@ -0,0 +1,78 @@
+import select
+import socket
+import traceback
+import logging
+
+from server.client import Client
+from server.messages import parse_data, ParseError, NormalMessage, PayloadMessage, ErrorMessage, UnknownCommand
+from server.messages import CMD_LOGIN, CMD_LOGOUT
+from server.models.user import InvalidUserCredentials
+
+
+log = logging.getLogger('loop')
```

Comment:
It's not an issue, it's just suggestion. Maybe it's better to store logger to class?
What is a common way for logging?

Reply:
yes, in this case it' better to move it to the class level